src/buffer.rs

```diff
@@ -660,6 +684,7 @@ impl Buffer {
         if let Some(path) = &self.path {
             path.write(&self.content())?;
             self.dirty = false;
+            self.owner = BufferOwner::User;
```

Comment:
If I did not misunderstand, this line is not tested, because this line is catered for this scenario, which is not tested: “Force Save a system-owned file that is unedited”

Reply:
Actually, that line should be removed. Good catch. It becomes owned by a BufferOwner::User on edit now, not on save. I'll remove the line now.

Reply:
That works too, I guess force saving an unedited file is too unconventional after all.